### ISSUE TITLE
Make piping Exa's output leave out the icons if you added --icons.

### ISSUE THREAD

**Issue by Oglo12** [ogham/exa#1193]
Hello, I have Exa aliased to ls, but I can't help but be bothered that piping `exa --icons` to another program includes the icons! I know that there is a way to fix this, after all, there is another Rust program called Bat that does this. Anyway, it would be really nice to have this fixed!

## Example when piping into FZF:
![image](https://github.com/ogham/exa/assets/80219532/f9dec021-6271-4121-95c4-75470e016266)

**Comment by eggbean** [comment]
Yes, it even adds icons with the `-1` switch, which is a bad oversight. I don't use `--icons` myself.

**Comment by kirll-lappo-dg7** [comment]
I think it would be better to make `--icons` option work the same as `--color` option.

For ex:

```bash
--colo[u]r=WHEN    when to use terminal colours (always, auto, never)
--icons=WHEN       when to use icons (always, auto, never)
```
To make it compatible, `auto` can be a default value when just `--icons` is specified.
